src/Components/Panels/PanelMenu.tsx

```diff
@@ -59,22 +59,6 @@ export class PanelMenu extends SceneObjectBase<PanelMenuState> implements VizPan
   constructor(state: Partial<PanelMenuState>) {
     super({ ...state, addExplorationsLink: state.addExplorationsLink ?? true });
     this.addActivationHandler(() => {
-      const viz = findObjectOfType(this, (o) => o instanceof VizPanel, VizPanel);
```

Comment:
We should expect that findObjectOfType doesn't find what we're looking for and account for that case. Similar to when we use `Array.find()`.

Reply:
Isn't the problem here that we do find something with findObjectOfType, it's just the wrong viz panel?

Reply:
looks like findObject will recursively check all of the children of each parent, so it grabs the viz panel for the logs volume instead